src/com/projectkorra/projectkorra/GeneralMethods.java

```diff
@@ -1653,23 +1657,22 @@ public static boolean isUndead(final Entity entity) {
 		if (entity == null) {
 			return false;
 		}
-		
+
 		switch (entity.getType()) {
-			case SKELETON:
-			case STRAY:
-			case WITHER_SKELETON:
-			case WITHER:
-			case ZOMBIE:
-			case HUSK:
-			case ZOMBIE_VILLAGER:
-			case PIG_ZOMBIE:

```

Comment:
Good that you removed `PIG_ZOMBIE` but it looks like you did not add [ZOGLIN](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/EntityType.html#ZOGLIN) or [ZOMBIFIED_PIGLIN](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/EntityType.html#ZOMBIFIED_PIGLIN) to this check.